pkg/vmimage/data/script.sh

```diff
@@ -42,8 +42,9 @@ cat >client-key.pem <<'EOF'
 {{ .Builder.ClientKey | PrivateKeyAsBytes | String }}
 EOF
 
-go get github.com/jim-minter/tlsproxy
-tlsproxy -insecure -key client-key.pem -cert client-cert.pem https://cdn.redhat.com/ &
+go get github.com/openshift/openshift-azure/cmd/azure
+set -x
+/go/bin/azure tlsproxy --insecure --key client-key.pem --cert client-cert.pem --hostname https://cdn.redhat.com/ &
```

Comment:
Can we avoid hardcoding this path and rely on `GOPATH` instead as we did previously? I don't see any need for this change.

```suggestion
azure tlsproxy --insecure --key client-key.pem --cert client-cert.pem --hostname https://cdn.redhat.com/ &
```